From 426b9e8d57c7fcaf1b8d780394898dad30131eba Mon Sep 17 00:00:00 2001
From: Rex Zhu <Rex.Zhu@amd.com>
Date: Mon, 17 Apr 2017 18:16:22 +0800
Subject: [PATCH 2757/3082] Revert "drm/amd/powerplay: set sensitivity of fan
 reaction to temperature changes."

Remove fan controller sensitivity until there is an encoding scheme
between SMC and PP.

This reverts commit 932f0d1fe341ee006e732f0aa1b1fe5320b4293d.
---
 drivers/gpu/drm/amd/powerplay/hwmgr/vega10_thermal.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_thermal.c b/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_thermal.c
index 4ada673..f4d77b6 100644
--- a/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_thermal.c
+++ b/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_thermal.c
@@ -553,8 +553,6 @@ int tf_vega10_thermal_setup_fan_table(struct pp_hwmgr *hwmgr,
 	if (!data->smu_features[GNLD_FAN_CONTROL].supported)
 		return 0;
 
-	table->FuzzyFan_PwmSetDelta = (uint16_t)hwmgr->thermal_controller.
-			advanceFanControlParameters.usFanOutputSensitivity;
 	table->FanMaximumRpm = (uint16_t)hwmgr->thermal_controller.
 			advanceFanControlParameters.usMaxFanRPM;
 	table->FanThrottlingRpm = hwmgr->thermal_controller.
-- 
2.7.4

